### ISSUE TITLE
Ideas for Possible New Reward Packages if this is possible using Decimal Numbers under Each option?

### ISSUE THREAD

**Issue by Conchipedia** [ClayC90/ClaysTKSite_Issues#18]
Package 1.1
/claim 1.1
20 Mortared Stone

Package 1.2
/claim 1.2
 20 Brick

Package 1.3
/claim 1.3
20 Glass

and so on..... (For ALL Building Blocks in order of Tiers)

Tree Seeds (Cedar,Birch and Acorn)

**Comment by ClayC90** [comment]
that will break the economy, but I guess I can create a table for this which will do conversions and such.

**Comment by Conchipedia** [comment]
was just an idea to give more choice for rewards as not everyone needs 20 logs

**Comment by ClayC90** [comment]
yeah I like the idea, but will have to include unit conversions so it's not just 20 iron bars for example, as they are worth more :P
